gpflow/ci_utils.py

```diff
@@ -18,7 +18,14 @@
 
 
 def is_continuous_integration():
-    return os.environ.get('CI', None) is not None
+    """
+    If the environment variable `DOCS` is set, always returns False.
+    Otherwise, returns `True` if notebooks are executed on CICD, else `False`.
```

Comment:
```suggestion
    Otherwise, returns `True` if notebooks are executed on CI, else `False`.
```
?

Reply:
continuous integration and continuous deployment